scripts/ingests/update_authorcodes.py

```diff
@@ -0,0 +1,40 @@
+from scripts.ingests.utils import *
+from sqlalchemy import select, or_
+import pandas as pd
+
+SAVE_DB = True  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+logger.setLevel(logging.INFO)
+
+if RECREATE_DB:
+    db.Publications.update().where(db.Publications.c.publication == 'Luyt79a').values(
+        publication='Luyt79.5').execute()
+    db.Publications.update().where(db.Publications.c.publication == 'Luyt79b').values(
+        publication='Luyt79.2').execute()
+
+#  creating a query that finds all the author codes that need to be updated
+stm = select([db.Publications.c.publication, db.Publications.c.bibcode]).where(
+    or_(db.Publications.c.publication.like('______a'), db.Publications.c.publication.like('______b'),
+        db.Publications.c.publication.like('______c'), db.Publications.c.publication.like('______d'),
+        db.Publications.c.publication.like('______e'), db.Publications.c.publication.like('______f'),
+        db.Publications.c.publication.like('______g'), db.Publications.c.publication.like('______h')))
+
+result = db.session.execute(stm)
+df = pd.DataFrame(result.all())
```

Comment:
BTW, I realized that this can also be compacted a little bit, particularly if you use the ORM way to call it (as opposed to select). For example, something like:
```
df = db.query(db.Publications.c.publication, db.Publications.c.bibcode). \
     filter(or_(db.Publications.c.publication.like('______a'), 
                db.Publications.c.publication.like('______b'), 
                etc)). \
     pandas()
```
But I'll approve regardless- no need to change unless you really want to.